opentera-webrtc-native-client/OpenteraWebrtcNativeClient/python/CMakeLists.txt

```diff
@@ -2,30 +2,31 @@ cmake_minimum_required(VERSION 2.6.4)
 
 project(opentera_webrtc_native_client)
 
-set(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR}/bin/${CMAKE_BUILD_TYPE})
+set(LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bin)
```

Comment:
Replaced with modern cmake version and removed build_type as it should already be included in `LIBRARY_OUTPUT_DIRECTORY` as mentioned here https://cmake.org/cmake/help/latest/prop_tgt/LIBRARY_OUTPUT_DIRECTORY.html